yarn-project/aztec.js/src/aztec_rpc_client.ts

```diff
@@ -32,7 +33,7 @@ export const createAztecRpcClient = (url: string, fetch = makeFetch([1, 2, 3], t
       NotePreimage,
       AuthWitness,
     },
-    { Tx, TxReceipt, L2BlockL2Logs },
+    { Tx, TxReceipt, L2BlockL2Logs, NodeInfo },
```

Comment:
Actually, do we need to add `NodeInfo` here? If we keep it as an interface instead of a type, and we initialise it as a plain object, doesn't the automagic (de)serialiser go into each field and serialise that, without having to register the class here?

If we can remove it and just keep NodeInfo as a type or interface (and not a class) we can roll back the new dependency on aztec/ethereum completely.